hwilib/devices/trezor.py

```diff
@@ -258,6 +258,26 @@ def ignore_input():
                     else:
                         raise BadArgumentError("Output is not an address")
 
+                # Add the derivation path for change, but only if there is exactly one derivation path
+                psbt_out = tx.outputs[i]
+                if len(psbt_out.hd_keypaths) == 1:
```

Comment:
just noting the ledger detection is more eager, trying any number of hd keypaths that appear to match. We could tighten that logic up in a follow-on PR.